mars/settings/base.py

```diff
@@ -125,5 +125,12 @@
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 
-STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
+STATIC_ROOT = os.path.join(BASE_DIR, "../staticfiles")
 STATIC_URL = "/static/"
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, "../frontend/static"),
+    os.path.join(BASE_DIR, "../frontend/static/templates"),
+    os.path.join(BASE_DIR, "../frontend/static/dist"),
```

Comment:
Is it django notation to provide these specific directories? Or could you have just provided `../frontend/static/**`?